api/python/quilt3/main.py

```diff
@@ -244,6 +248,11 @@ def create_parser():
     )
     verify_p.set_defaults(func=cmd_verify)
 
+    # version
+    shorthelp = "Display the version of quilt3"
+    version_p = subparsers.add_parser("version", description=shorthelp, help=shorthelp, allow_abbrev=False)
```

Comment:
@armandmcqueen thanks we needed this. can you make ti `--version` so it's more consistent with our friends:
```
apk-mbp-3:catalog karve$ git --version
git version 2.17.2 (Apple Git-113)
apk-mbp-3:catalog karve$ docker --version
pDocker version 19.03.5, build 633a0ea
apk-mbp-3:catalog karve$ python --version
Python 2.7.16
```

Reply:
Fixed